index.js

```diff
@@ -180,6 +181,7 @@ function transformPhrase(phrase, substitutions, locale, tokenRegex) {
 
   var result = phrase;
   var interpolationRegex = tokenRegex || defaultTokenRegex;
+  var pluralRulesOrDefault = pluralRules || defaultPluralRules;
```

Comment:
this means that you have to supply both, or neither - you can’t just override one. Is that intended?

Reply:
Yes, both values are fairly tightly coupled, so allowing them to be set individually would increase the scope of changes that would be breaking.  Allowing the values to be set a la carte doesn't substantially improve usability for the case of adding a locale to an existing plural rule (this would require a different API).

This allows for configuration without exposing the specifics of the defaults. The disadvantage is that if you wish to override, you must fully override the values, but that matches my use case anyway.